R/use_lintr.R

```diff
@@ -43,5 +43,29 @@ use_lintr <- function(path = ".", type = c("tidyverse", "full")) {
     )
   )
   write.dcf(the_config, config_file, width = Inf)
+
+  # If this is an R package and if available, add .lintr file to .Rbuildignore
+  if (file.exists("DESCRIPTION") && file.exists(".Rbuildignore")) {
```

Comment:
This test looks in the current working directory, not where the config was placed.

Reply:
Since there is no guarantee that `DESCRIPTION` and `.lintr` will be placed in the same directory, I believe that the current working directory can should be used.